portal/src/theme.tsx

```diff
@@ -0,0 +1,65 @@
+import { createTheme } from "@fluentui/react/lib/Styling";
```

Comment:
I suppose we should import from `@fluentui/react` instead.

Reply:
[This is a pattern recommended by them](https://github.com/microsoft/fluentui/tree/master/packages/styling#using-the-styling-package)

Reply:
I think they recommend against importing `@fluentui/styling` directly, not importing from `@fluentui/react`, since the [documentation](https://developer.microsoft.com/en-us/fluentui/#/controls/web/themes) is using this pattern.